.github/workflows/docs.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: 3.x
-      - run: pip install mkdocs-material
+      - run: pip install mkdocs-material mkdocs-redirects
```

Comment:
the `LSP/docs/README.md` also needs to be updated to include `mkdocs-redirects`:
```
Install Material Theme - https://squidfunk.github.io/mkdocs-material/getting-started/#installation

pip install mkdocs mkdocs-material mkdocs-redirects

```